package.json

```diff
@@ -24,6 +25,7 @@
     "cors": "^2.8.5",
     "cross-fetch": "^3.1.5",
     "dotenv": "^16.0.1",
+    "ethers": "5.6.2",
```

Comment:
It's better to use just the dependency we need, I guess here is `@ethersproject/wallet`